uber/api.py

```diff
@@ -664,7 +664,7 @@ def jobs(self, department_id):
                 'is_shiftless': True,
                 'is_setup_approval_exempt': True,
                 'is_teardown_approval_exempt': True,
-                'max_consecutive_hours': 0,
+                'max_consecutive_hours': True,
```

Comment:
Not sure why this was set to zero -- I think I vaguely recall that this field was added later, so it was probably for backwards compatibility. We definitely want to grab the value going forward.